core/templates/dev/head/domain/question/QuestionObjectFactory.js

```diff
@@ -48,6 +48,10 @@ oppia.factory('QuestionObjectFactory', ['StateObjectFactory',
       return this._version;
     };
 
+    Question.createDefaultQuestionData = function() {
+      return StateObjectFactory.createDefaultState(null);
```

Comment:
Perhaps cleaner API would be createDefaultQuestionState(); -- it could delegate to createDefaultState() behind the scenes. In this context the `null` doesn't have an obvious meaning.

Reply:
Done.